### PR TITLE
chore: add fun coverage badge

### PR DESCRIPTION
**Pull request by kyranjamie**
> [Download the latest build](https://github.com/blockstack/stacks-wallet/actions/runs/884990727)

Wanted to add the mega super fun generated coverage badge, as per web wallet

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -19,16 +19,15 @@ module.exports = {
...
   globals: {
     api: true,
     CONFIG: true,
     'ts-jest': {
       tsconfig: 'tsconfig.tests.json',
       diagnostics: {
         ignoreCodes: [
-          6133,
-          2304,
+          6133, 2304,
```
Comment (fbwoolf) [inline jest.config.js:30]: Did you mean to move these on the same line?

**Review by fbwoolf — APPROVED** [review]

**Comment by aulneau** [comment]
giphy (1)

File: .github/workflows/gh-pages.yml [hunk 2]
```diff
@@ -42,6 +42,9 @@ jobs:
       - name: Run jest
         run: rm -rf docs/coverage && yarn test --coverage
 
+      - name: Make coverage badge
+        run: npx make-coverage-badge
```
Comment (aulneau) [inline .github/workflows/gh-pages.yml:46]: would it be safer to use a specific tagged version rather than relying on the latest?

**Review by aulneau — APPROVED** [review]

File: jest.config.js (hunk 1 above)
Comment (kyranjamie) [reply jest.config.js:30]: This is owing to prettier change from the other PR

**Review by CharlieC3 — APPROVED** [review]
:shipit:

File: .github/workflows/gh-pages.yml (hunk 2 above)
Comment (kyranjamie) [reply .github/workflows/gh-pages.yml:46]: Probably safer, but would only break the badge—will do if it breaks

**Pull request merged**